docs/platforms/dotnet/common/configuration/msbuild.mdx

```diff
@@ -186,6 +193,25 @@ By default, when `SentrySetCommits` is enabled, the SDK will set commits with th
 
 </ConfigKey>
 
+<ConfigKey name="SentryUploadAndroidProGuardMapping" supported={["dotnet.maui"]}>
+
+Controls uploading ProGuard mapping file to Sentry after the build. Defaults to `false` (disabled).
+
+Set to `true` to upload ProGuard mapping file to Sentry.
+Your resulting APK will be injected with the metadata `io.sentry.proguard-uuid`. `AndroidManifest.xml` will contain a metadata entry such as the following:
+
+```xml
+<meta-data
+    android:name="io.sentry.proguard-uuid"
+    android:value="YOUR-UUID-HERE" />
+```
+
+Do not overwrite `io.sentry.proguard-uuid` metadata directly. See [SentryProGuardUUID](#SentryProGuardUUID) to set your own ProGuard UUID.
+
+`UseSentryCLI` must be set to `true` for the ProGuard mapping upload to work.
```

Comment:
Is this correct? It looks like setting the `SentryUploadAndroidProGuardMapping` automatically/implicitly enables `UseSentryCLI`:
https://github.com/getsentry/sentry-dotnet/blob/b2b8d492fe62a483200e4564eafffb0ae1cf7d08/src/Sentry/buildTransitive/Sentry.targets#L69-L73